src/tests/regressionTests.test.tsx

```diff
@@ -1054,6 +1054,28 @@ describe("regression tests", () => {
     expect(API.getSelectedElements()).toEqual(selectedElements_prev);
   });
 
+  it("deleting last but one element in editing group should unselect the group", () => {
+    const rect1 = UI.createElement("rectangle", { x: 10 });
```

Comment:
Added this test due to regression introduced by initially incorrectly rewritten `selectGroups`.